### ISSUE TITLE
List Views, Reports, Field Values

### ISSUE THREAD

**Issue by JodieM** [SFDO-Community/Salesforce-Indicators#63]
I want a way to show the Indicators in Field values so we can use the same CMDT setup but then show them in a field that can be visible in a report, list view, or on a record. 

My first thoughts are an Invocable Apex that returns a value that can be placed into a field via Flow. 
But Lightning Avatar does not have a way to output from Apex. 
So do we simplify and have only the Text Value or Emoji to return via the Invocable Apex. 

So maybe - an invocable Apex that takes in a Bundle Name and then loops through the Bundle Items and returns only the text values and separates them by a customisable separator (like the Concatenation option in DLRS). That might work for a simple use-case for now.

**Comment by nzgonjanin** [comment]
I may help with invocable method, but will need to fully understand objectives.

**Comment by tschug** [comment]
I do not believe we can support Reports or List Views, since they do not display any LWCs.

We may want to change this Issue's title to be "Flow Support" or create a new one with it being scoped accordingly.

**Comment by JodieM** [comment]
OK, I am going to close this, because a) it's not really doable, and b) it could be potentially done with creating an Icon in the UnofficialSF Data Table component and displaying the data table in a flow on a page. This can be achieved by creating a formula field that returns a text value of the icon name, and then using that field to display the icon.
